module/input/NatNet/src/NatNet.hpp

```diff
@@ -26,6 +26,20 @@
 namespace module::input {
 
     class NatNet : public NUClear::Reactor {
+    private:
+        /// @brief Stores configuration values
+        struct Config {
+            Config() = default;
+            /// @brief Motive Multicast Address
+            std::string multicast_address = "";
+            /// @brief UDP Command port for communicating with motive
+            uint32_t command_port = 0;
+            /// @brief UDP Data port for communicating with motive
+            uint32_t data_port = 0;
+            /// @brief Dump packets configuraton settings
```

Comment:
No need to say this is a configuration setting. It's in the config struct.
When documenting describe what something does, not what it is. You can see what it is because you're looking at it :P